packages/main/src/Dialog.js

```diff
@@ -212,10 +215,43 @@ class Dialog extends Popup {
 	}
 
 	onExitDOM() {
-		this._dragMouseMoveHandler = null;
-		this._dragMouseUpHandler = null;
+		delete this._screenResizeHandler;
+		delete this._dragMouseMoveHandler;
+		delete this._dragMouseUpHandler;
+		delete this._resizeMouseMoveHandler;
+		delete this._resizeMouseUpHandler;
+	}
```

Comment:
is it really needed to delete the handlers? Can't we let them be and initialize them in the constructor()?

Reply:
Deleting isn't really needed, you're right. With regards to when should the handlers be initialized, I want to delay this until the web component is going to be rendered on the page. Contructors get called before connectedCallback which is too early. Here, connectedCallback is onEnterDOM.